src/otx/algorithms/classification/adapters/mmcls/models/classifiers/supcon_classifier.py

```diff
@@ -12,9 +12,10 @@
 class SupConClassifier(ImageClassifier):
     """SupConClassifier with support for classification tasks."""
 
-    def __init__(self, backbone, neck=None, head=None, pretrained=None, **kwargs):
+    def __init__(self, backbone, neck=None, head=None, pretrained=None, task_adapt=None, **kwargs):
         self.multilabel = kwargs.pop("multilabel", False)
         self.hierarchical = kwargs.pop("hierarchical", False)
+        self.task_adapt = task_adapt
```

Comment:
Is `task_adapt` used in this class?

Reply:
No, but it is for unification between classifier in OTX, and also I think the classifier will need this when it supports incremental learning